src/OpenColorIO/ops/Lut3D/Lut3DOpData.h

```diff
@@ -0,0 +1,180 @@
+/*
+Copyright (c) 2018 Autodesk Inc., et al.
+All Rights Reserved.
+
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are
+met:
+* Redistributions of source code must retain the above copyright
+  notice, this list of conditions and the following disclaimer.
+* Redistributions in binary form must reproduce the above copyright
+  notice, this list of conditions and the following disclaimer in the
+  documentation and/or other materials provided with the distribution.
+* Neither the name of Sony Pictures Imageworks nor the names of its
+  contributors may be used to endorse or promote products derived from
+  this software without specific prior written permission.
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+"AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+*/
+
+#ifndef INCLUDED_OCIO_LUT3DOPDATA_H
+#define INCLUDED_OCIO_LUT3DOPDATA_H
+
+#include <vector>
+
+#include <OpenColorIO/OpenColorIO.h>
+
+#include "Op.h"
+#include "ops/OpArray.h"
+
+OCIO_NAMESPACE_ENTER
+{
+class Lut3DOpData;
+typedef OCIO_SHARED_PTR<Lut3DOpData> Lut3DOpDataRcPtr;
+
+class Lut3DOpData : public OpData
+{
+public:
+
+    // Enumeration of the inverse 3D LUT styles.
```

Comment:
Following up on the working group meeting today, please delete line 48 and replace with:

// There are two inversion algorithms provided for Lut3D, an exact method (that assumes use of tetrahedral in the forward direction) and a fast method that bakes the inverse out as another forward Lut3D.  The exact method is currently unavailable on the GPU.  Both methods assume that the input and output to the Lut3D are roughly perceptually uniform.  Values outside the range of the forward Lut3D are clamped to someplace on the exterior surface of the Lut3D.